cmd/di.go

```diff
@@ -97,7 +97,7 @@ type Dependencies struct {
 	ConnectionManager  connection.Manager
 	ConnectionRegistry *connection.Registry
 
-	ServiceRunner         *service.Runner
+	ServiceManager        *service.Manager
```

Comment:
```suggestion
	ServicesManager        *service.Manager
```
Since now we manage multiple services, not a singe service.

Reply:
In general, I would leave ServiceManager, but since it is ambiguous with previous ServiceManager managing just a single service, maybe we should make it plural.

Reply:
Renamed